test/test/lock/LockTests.java

```diff
@@ -0,0 +1,17 @@
+package test.lock;
+
+import one.profiler.test.Output;
+import one.profiler.test.Test;
+import one.profiler.test.TestProcess;
+
+public class LockTests {
+
+    @Test(mainClass = DatagramTest.class, debugNonSafepoints = true)
+    public void datagramSocketLock(TestProcess p) throws Exception {
+        Output out = p.profile("-e cpu -d 3 -o collapsed --cstack dwarf"); 
+        out.assertRatioGreater("(Unsafe..ark|Unsafe..npark|PlatformEvent::.ark|PlatformEvent::.npark)", 0.1);
```

Comment:
`Unsafe..ark` matches both Java method `Unsafe.park` and a native function `Unsafe_Park`. The test intends to check the presence of the native one. I know it fail on Alpine now, but that's fine - it's a real issue that needs to be fixed.